test/asynchronous/test_collection.py

```diff
@@ -2258,9 +2251,16 @@ async def test_helpers_with_let(self):
         for let in [10, "str", [], False]:
             for helper, args in helpers:
                 with self.assertRaisesRegex(TypeError, "let must be an instance of dict"):
-                    await helper(*args, let=let)  # type: ignore
+                    # c.find() is synchronous, can't await it
```

Comment:
It could be cleaner here to do this:
```python
async def afind(*args, **kawrgs):
    return c.find(*args, **kwargs)
    
helpers = [
...
            (afind, ({})),

]
```
That way we can always await here.

Reply:
I like it, done.